scripts/test/test_install.ps1

```diff
@@ -49,7 +49,7 @@ foreach ($package in $packages) {
 }
 
 
-$exclude_tests = @("installer.vm")
+$exclude_tests = @("installer.vm", "idapro.vm")
```

Comment:
Note to test the `idapro.vm` package locally, this needs to be modified. The alternative would be to edit the workflows that test packages to exclude it there, but it is tricky to keep them consistent. I suggest we keep it like this until it is clear what happens with IDA free 9.0, in case the packages can be soon combined and then tested.